place/views.py

```diff
@@ -1,27 +1,181 @@
+                )
+                return JsonResponse({'message':'SUCCESS'},status=200)
+        
+        except User.DoesNotExist:
+            return JsonResponse({'message':'USER_DOSENOTEXIST'},status=401)
+        
+        except Place.DoesNotExist:
+            return JsonResponse({'message':'PLACE_DOSENOTEXIST'},status=401)
+    
+    def get(self,request,place_id):
+        try:
+            # data = json.loads(request.body)
+            ratings = Rating.objects.filter(place_id=place_id)
+            reviews = []
+            
+            for rate in ratings:
+                reviews.append({
+                    "rate_id"  :rate.id, 
+                    "starpoint":rate.starpoint,
+                    "user_name":rate.user.name,
+                    "created_at":rate.created_at,
+                    "comments":rate.comment,
+                })
+            return JsonResponse({'message':'SUCCESS','place':reviews},status=200)
+        
+        except Place.DoesNotExist:
+            return JsonResponse({'message':'PLACE_DOSENOTEXIST'},status=401)
+  
+class ReviewView(View):
+    @checkAuthDecorator 
+    def delete(self,request,rating_id):
+        try:
+            Rating.objects.filter(id=rating_id).delete()
+            return JsonResponse({'message':f'id : {rating_id} 삭제'},status=200)
+        
+        except Rating.DoesNotExist:    
+            return JsonResponse({'message':'RATING_DOSENOTEXIST'},status=401)
+    @checkAuthDecorator 
+    def put(self,request,rating_id):
+        try:
+            review_info = Rating.object.get(id=rating_id)
+            review_info( 
+                    starpoint = data['starpoint'],
+                    comments = data['comment']
+            ).save()
+        except Exception as ex:
+            return JsonResponse({'message':f'ex'},status=401)
+
+class SearchView(View):
+    # 검색기능
+    def post(self,request):
+        data = json.loads(request.body)
+        search_info = []
+        places = Place.objects.filter(
+            Q(title__contains = data['search']) |
+            Q(description__contains = data['search']) |
+            Q(region__name__contains = data['search']) |
+            Q(category__name__contains = data['search'])
+            ).all()
+        for place in places:
+            ratings = place.related_rating_place.all()
+            for rating in ratings:
+                result=[]
+                result.append(rating.starpoint)    
+                sum_rating = sum(result)/ratings.count()
+            
+            search_info.append({
+                'place_id'      : place.id,
+                'category'      : place.category.name,
+                'title'         : place.title,
+                'region'        : place.region.name,
+                'starpoint_avg' : sum_rating
+                })
+        
+        return JsonResponse({'search':f'{search_info}'},status=200)
+
+class PlaceDetailView(View):
+    # Place 모든 정보
+    def get(self,request,place_id):
```

Comment:
```suggestion
    def get(self, request, place_id):
```
함수에 선언에 들어가는 인자들은 한 칸씩 간격을 넣어주시는 것이 좋습니다